src/index.tsx

```diff
@@ -25,11 +25,7 @@ Modal.setAppElement('#root');
 const container = document.getElementById('root') as Element;
 const root = createRoot(container);
 
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+root.render(<App />);
```

Comment:
It could be somehow mentioned why the StrictMode was removed. I think we want it back when possible.

"HDS Login Component is too aggressive with the login time issues and the StrictMode gives unwanted issues."

Reply:
fixed